src/App.js

```diff
@@ -1,30 +1,29 @@
 import React, { Component } from 'react';
+
 import 'semantic-ui-css/semantic.min.css';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
 
 import Login from './components/Authentication/LoginDummy';
 import Error from './components/ErrorHandlers/MissingPageError';
 import Home from './Home';
 import Header from './components/Header/Header';
+import ArticleContainer from './components/Articles/ArticlesContainer';
+
 
 class App extends Component {
   render() {
     return (
-
-      <BrowserRouter>
-        <div>
-          <div className="App">
-            <Header />
-          </div>
+      <React.Fragment>
+        <Header />
+        <BrowserRouter>
           <Switch>
-            {/* exact looks for the exact path,
-            it prevents the home page from being rendered in all the pages */}
             <Route path="/" component={Home} exact />
```

Comment:
use `exact` in your routes. Also since the articles page is your landing page change the index page to point to it.